src/mmw/apps/modeling/tasks.py

```diff
@@ -176,13 +176,42 @@ def run_tr55(census, model_input):
     et_max = 0.207
     precip = model_input['precip']
 
+    # TODO: These next two lines are just for
+    # demonstration purposes.
+    modifications = get_census_modifications(model_input)
+    census['modifications'] = modifications
+
     def simulate_day(cell, cell_count):
         soil_type, land_use = cell.lower().split(':')
         et = et_max * lookup_ki(land_use)
         return simulate_cell_day((precip, et), cell, cell_count)
 
     model_output = simulate_modifications(census, fn=simulate_day)
+
     return {
         'runoff': model_output,
         'quality': format_quality(model_output)
     }
+
+
+# TODO: For demonstration purposes.
+def get_census_modifications(model_input):
+    """
+    For each modification that was drawn, replace
+    a cell of commercial with a cell of
+    chaparral. This has the effect of improving
+    infiltration and evapotranspiration.
+    """
+    count = len(model_input['modifications'])
```

Comment:
Didn't see the recent comment about removing mods, that may have been what I was seeing re: lack of change.